libs/ui/src/tokens/components/_form-control.css

```
@@ -0,0 +1,71 @@
+  /* Placeholder */
+  --color-form-control-placeholder: var(--color-fg-placeholder);
+
+  /*
+   * Shared heights for sm/md form controls + buttons.
+   * Formula: border×2 + button-padding-y×2 + 1lh
+   * (1lh resolves to the element's computed line-height at usage time)
+   * Apply h-form-control-sm / h-form-control-md to both buttons and
+   * form-control wrappers so they share an identical rendered height.
+   */
+  --height-form-control-sm: calc(
+    var(--border-width-form-control) *
```

Stylelint (17.3.0)

[error] Unexpected newline after "*" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)